client/src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro'
+
+const MovieCard = ({ imgUrl, title, year, genre }) => {
+  return (
+    <Wrapper>
+      <img src={imgUrl} alt={title} height="562" width="375" />
+      <h2>{title}</h2>
+      <p>
+        {year} - {genre}
+      </p>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
+  background-color: var(--color-yellow);
+  border: 2px solid var(--color-lila);
+  border-radius: 10px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0.5rem auto;
+  padding: 0.5rem;
+  width: 340px;
+
+  img {
+    height: 200px;
+    width: 133px;
```

Comment:
Why do you set a different height and width here than in the `img` tag as attributes? I would recommend reading this article about layout shifts: https://gideongoddard.co.uk/blog/why-you-should-always-add-width-and-height-attributes-to-your-images-again/